src/main/webapp/resources/celJS/contextMenu/contextmenu.js

```diff
@@ -19,61 +19,50 @@
  */
 
 /* HELPERS */
-var cmDefaultItems = new Array();
-var cmOutliner = null;
+const cmDefaultItems = new Array();
+let cmOutliner = null;
 
-function contextMouseOver(n){
-  classAttribute = n.getAttributeNode('class');
+const contextMouseOver = function(n) {
+  const classAttribute = n.getAttributeNode('class');
   classAttribute.nodeValue = "contextMenuLinkOver";
 
-  $(document).stopObserving('mousedown', myContextMenu.hide);
+  document.removeEventListener('mousedown', myContextMenu.hideBind);
   return true;
-}
+};
 
-function contextMouseOut(n){
-  classAttribute = n.getAttributeNode('class');
+const contextMouseOut = function(n) {
+  const classAttribute = n.getAttributeNode('class');
   classAttribute.nodeValue = "contextMenuLink";
   
-  $(document).observe('mousedown', myContextMenu.hide);
+  document.addEventListener('mousedown', myContextMenu.hideBind);
```

Comment:
the bind `hideBind` shouldn't be necessary with arrow functions:
```suggestion
  document.addEventListener('mousedown', e => myContextMenu.hide(e));
```
can be replaced everywhere like so